docs/roadmap/rm-2020.md

```diff
@@ -0,0 +1,79 @@
+# 2020 Eventing Roadmap
+
+## References
+
+[Personas and User Stories](../personas.md):
+
+Knative Eventing caters to the following personas when developing different
+features
+
+1. Event consumer (developer)
+1. Event producer
+1. System Integrator
+1. Contributors
```

Comment:
I think that is perhaps a thing on me. But to me that persona sound artificial :-) 

When I use Knative (Eventing), I am in one of the first three roles - or all.

But when I send a fix, b/c I use it, I am also a contributior.   I mean ... if I not use it, why would I contribute ? :-) 

I know that these personas are already defined - but feel IMO a bit... artificial ... :-)

Reply:
A better definition of the personas would clearly help in justifying why a feature is needed or not (among other things). +1 on expanding/redefining personas.